app/models/user.rb

```diff
@@ -15,4 +15,8 @@ def permissions
   def departments
     Department.where(slug: department_slugs)
   end
+
+  def foo
```

Comment:
HA!  Left over debugging from a blog post I was writing. Thanks for catching this!